data/drupal/widgets.graphql

```diff
@@ -1,5 +1,7 @@
 #import "./media.graphql"
 #import "./taxonomy.graphql"
+#import "./blocks.graphql"
```

Comment:
Are these commented out imports needed?

Reply:
We use a the webpack graphql plugin which is what allows imports in graphql files but because imports aren't actually apart of graphql it uses comments to do it so that it maintains compatibility.

Reply:
Ohhh, I see. That resolves it for me.